### PR TITLE
Replace buildferry with createbob

### PR DESCRIPTION
**Pull request by gunchleoc**
One less program for modders to learn ;)

### CODE REVIEW

File: src/logic/map_objects/tribes/worker_program.cc [hunk 1]
```diff
@@ -722,7 +730,8 @@ buildferry
 ^^^^^^^^^^
 .. function:: buildferry
 
-   Adds a new instance of this tribe's ferry to the map at the worker's current location. Example::
+   **DEPRECATED** Adds a new instance of this tribe's ferry to the map at the worker's current
+location. Example::
```
Comment (Noordfrees) [inline src/logic/map_objects/tribes/worker_program.cc]: Just replace the whole comment and usage example with a line "DEPRECATED, use `createbob=TRIBENAME_ferry`" instead

Comment (gunchleoc) [reply src/logic/map_objects/tribes/worker_program.cc]: Done

**Review by Noordfrees — APPROVED** [review]
LGTM, not tested

**Comment by gunchleoc** [comment]
I have tested all ferries and the gamekeeper.

**Pull request merged**